source/sxl_traffic_light_controller.rst

```diff
@@ -163,35 +203,47 @@ Dead lock error |br|  |br| Used for dead lock errors. |br|  |br| For instance; a
    :loc: H
    :spec: >{\raggedright\arraybackslash}p{0.15\linewidth} p{0.08\linewidth} p{0.13\linewidth} p{0.35\linewidth}
 
-   ========  =======  =============  =================
-   Name      Type     Value          Comment
-   ========  =======  =============  =================
-   timeplan  integer  [designation]  Current time plan
-   ========  =======  =============  =================
+   ========  =======  =======  =================
+   Name      Type     Value    Comment
+   ========  =======  =======  =================
+   timeplan  integer  [1-255]  Current time plan
+   ========  =======  =======  =================
 ..
 
 A0009
 ^^^^^
 
-Other error |br|  |br| Used for other errors not covered by any other alarm type |br|  |br| Is a "minor fault" defined according to 3.11 in EN12675.
+Other error
+
+Used for other errors not covered by any other alarm type
+
+Is a "minor fault" defined according to 3.11 in EN12675.
 
 
 A0010
 ^^^^^
 
-Door open |br|  |br| Used for open door (room or cabinet).
+Door open
+
+Used for open door (room or cabinet).
```

Comment:
The TLC isn't necessarily placed in a cabinet. It can be placed in a dedicated room in a basement of a nearby house or in a small building. This is refered to as a "apparaturm" or "teknikhus" in swedish. (technical room/building)

Reply:
ok i see